debian/cephadm.postinst

```diff
@@ -25,7 +25,7 @@ case "$1" in
        # 1. create user if not existing
        if ! getent passwd | grep -q "^cephadm:"; then
          echo -n "Adding system user cephadm.."
-         adduser --quiet --system --disabled-password --gecos 'Ceph-dameon user for cephadm' --shell /bin/bash cephadm 2>/dev/null || true
+         adduser --quiet --system --disabled-password --gecos 'Ceph-daemon user for cephadm' --shell /bin/bash cephadm 2>/dev/null || true
```

Comment:
```suggestion
         adduser --quiet --system --disabled-password --gecos 'cephadm user for mgr/cephadm' --shell /bin/bash cephadm 2>/dev/null || true
```



Make this the same on all platforms (rhel, suse, debian)?

This is what's used in the rpm spec:
https://github.com/ceph/ceph/blob/71f75fe41f84a0a255dcb6d607096dea2f83eb1d/ceph.spec.in#L1457

cc @sebastian-philipp